src/smoothscroll.js

```diff
@@ -20,6 +20,9 @@ function isMicrosoftBrowser(userAgent) {
   return new RegExp(userAgentPatterns.join('|')).test(userAgent);
 }
 
+var ALLOWED_BEHAVIOR_VALUES = [undefined, 'auto', 'instant', 'smooth'];
+var ALLOWED_BLOCK_VALUES = [undefined, 'start', 'end'];
```

Comment:
https://drafts.csswg.org/cssom-view/#dictdef-scrollintoviewoptions
https://drafts.csswg.org/cssom-view/#enumdef-scrolllogicalposition

It is better to add the options of "center"  and "nearest".